### PR TITLE
Change PC Operating Systems to Introductory Operating Systems

### PR DESCRIPTION
**Pull request by dngray**
https://deploy-preview-1995--privacytools-io.netlify.app/operating-systems/#os

### CODE REVIEW

**Comment by jonaharagon** [comment]
We should change the navbar menu and add an entry there for the advanced section as well so it matches the page.

**Comment by Mikaela** [comment]
What does Introductory mean in this context?

**Comment by dngray** [comment]
> What does Introductory mean in this context?

A place where you should start.

**Comment by dngray** [comment]
> We should change the navbar menu and add an entry there for the advanced section as well so it matches the page.

I decided we don't really need to link to introductory/advanced as they're both "PC OS", so just link to the main page.

If we did add those two separate subcategories to the nav.html, we'd need to think of suitable icons for each.

File: _includes/nav.html [hunk 1]
```diff
@@ -86,7 +86,7 @@
             </span>
           </summary>
           <span class="nav-dropdown">
-            <a class="dropdown-item" href="/operating-systems/#os"><span class="fas fa-th-large fa-fw"></span> PC OS</a>
+            <a class="dropdown-item" href="/operating-systems"><span class="fas fa-th-large fa-fw"></span> PC OS</a>
```
Comment (jonaharagon) [inline _includes/nav.html:89]: What is the reason for this change?
